k8s/zookeeper/chart/zookeeper/templates/application.yaml

```diff
@@ -32,11 +32,11 @@ spec:
       gcloud container clusters get-credentials [CLUSTER_NAME] --zone [CLUSTER_ZONE]
       ```
 
-      Where:
+      where:
       * `[CLUSTER_NAME]` is the name of the cluster for this application
-      * `[CLUSTER_ZONE]` is the zone, in which the cluster is operating
+      * `[CLUSTER_ZONE]` is the zone in which the cluster is operating
 
-      Please read [here](https://cloud.google.com/kubernetes-engine/docs/quickstart#get_authentication_credentials_for_the_cluster) for more information.
+      Further documentation about getting authentication credentials is available [here](https://cloud.google.com/kubernetes-engine/docs/quickstart#get_authentication_credentials_for_the_cluster) for more information.
```

Comment:
Does this sentence make sense?

Further documentation about getting authentication credentials is available here for more information.

Reply:
I believe it does, even if it could be: Further documentation about getting authentication credentials is available here.

Reply:
Good catch, @wgrzelak - fixing that now! Using @khajduczenia 's version.